### ISSUE TITLE
Packager error "Executable not found"

### ISSUE THREAD

**Issue by Leaf13** [TurboWarp/packager-extras#10]
# A problem: error was returned after selecting the name and location of the installer
![Screenshot 2021-10-10 105021](https://user-images.githubusercontent.com/61076957/136676605-7f1d554f-3b48-4252-8d1b-3faa7b31c7df.png)

**Comment by GarboMuffin** [comment]
Are you sure you selected the .zip file spit out by the packager website?

Did your antivirus display any warnings when you got that message?

**Comment by Leaf13** [comment]
Yes, I did select the zip and no, my av did not display warnings

**Comment by GarboMuffin** [comment]
What environment is the packager set to?

**Comment by Leaf13** [comment]
Windows

**Comment by Leaf13** [comment]
What I think is going wrong is that it didn't unzip it properly so the temp file was blank so it couldn't find something it needed in the zip file
